packages/config/tests/parse/fixtures/parse_backslash_valid/netlify.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+ignore = "test \\[this\\]\ntest \" \b \t \n \f \r \u0000 \U00000000 \
```

Comment:
GitHub TOML syntax highlighting is not happy with a backslash followed by newline. But this is explicitly allowed by the TOML specification: https://toml.io/en/v1.0.0#string